tests/integration/vlan_test.py

```diff
@@ -90,9 +73,60 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[Interface.KEY]
 
 
+def test_two_vlans_on_eth1_change_mtu(eth1_up):
+    desired_state = statelib.show_only(('eth1',))
+    eth1_state = desired_state[Interface.KEY][0]
+    vlans_state = create_two_vlans_state()
+    desired_state[Interface.KEY].extend(vlans_state[Interface.KEY])
+    for iface in desired_state[Interface.KEY]:
+        iface[Interface.MTU] = 2000
+    libnmstate.apply(desired_state)
+
+    eth1_102_state = next(
+        ifstate
+        for ifstate in desired_state[Interface.KEY]
+        if ifstate[Interface.NAME] == VLAN2_IFNAME
+    )
+    eth1_state[Interface.MTU] = 2200
+    eth1_102_state[Interface.MTU] = 2200
+    libnmstate.apply(desired_state)
+
+    eth1_vlan_iface_cstate = statelib.show_only((VLAN_IFNAME,))
+    assert eth1_vlan_iface_cstate[Interface.KEY][0][Interface.MTU] == 2000
+
+
+def test_two_vlans_on_eth1_change_base_iface_mtu(eth1_up):
+    desired_state = statelib.show_only(('eth1',))
+    eth1_state = desired_state[Interface.KEY][0]
+    vlans_state = create_two_vlans_state()
+    desired_state[Interface.KEY].extend(vlans_state[Interface.KEY])
+    for iface in desired_state[Interface.KEY]:
+        iface[Interface.MTU] = 2000
+    libnmstate.apply(desired_state)
+
+    eth1_state = statelib.show_only(('eth1',))
```

Comment:
Why do you need to read it again? You already had it once.

Reply:
Oops, fixed.